recipes/platform.hashing/all/conanfile.py

```diff
@@ -48,22 +48,39 @@ def validate(self):
                 self.name, self.settings.compiler))
 
         elif tools.Version(self.settings.compiler.version) < minimum_version:
-            raise ConanInvalidConfiguration("platform.Hashing/{} "
-                                            "requires C++{} with {}, "
-                                            "which is not supported "
-                                            "by {} {}.".format(
-                self.version, self._minimum_cpp_standard, self.settings.compiler, self.settings.compiler,
+            raise ConanInvalidConfiguration("{}/{} requires c++{}, "
+                                            "which is not supported by {} {}.".format(
+                self.name, self.version, self._minimum_cpp_standard, self.settings.compiler,
                 self.settings.compiler.version))
 
         if self.settings.compiler.get_safe("cppstd"):
             tools.check_min_cppstd(self, self._minimum_cpp_standard)
 
+        if self.settings.arch in ("x86", ):
+            raise ConanInvalidConfiguration("{} does not support arch={}".format(self.name, self.settings.arch))
```

Comment:
If you meant to move this to `package_info`, then this does not matter.
`validate` is called both when creating the `platform.hashing` package + when it is used by dependencies.

Reply:
I just like the `package...` methods at the end 😋